modules/ROOT/pages/configure-partner.adoc

```diff
@@ -21,18 +21,22 @@ To create a new partner, you need the following information:
 [[create-partner]]
 == Create a Partner
 
+=== Prerequisite
```

Comment:
I know that there is only one prereq, but I would still keep the heading as Prerequisites.